gui/checkthread.cpp

```diff
@@ -117,10 +117,11 @@ void CheckThread::check(const Settings &settings)
     start();
 }
 
-void CheckThread::analyseWholeProgram(const QStringList &files)
+void CheckThread::analyseWholeProgram(const QStringList &files, const QString& ctuInfo)
 {
     mFiles = files;
     mAnalyseWholeProgram = true;
+    mCtuInfo = ctuInfo;
```

Comment:
Since this is always converted to a `std::string` on usage I think it makes sense to store at as such if we are copying it anyways.

Reply:
done.